aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -134,6 +140,11 @@ module AptosFramework::Stake {
         mint_cap: MintCapability<TestCoin>,
     }
 
+    struct ValidatorPerformance has key {
+        num_blocks: u64,
```

Comment:
different validator can have different number of blocks ? eg: a validator can participate in 100 voting while another can participate in 2 voting. Should we track their lifetime votes individually?

Reply:
No that's not possible. A validator can only join and leave at epoch boundary. So even if they request to leave the validator set and the epoch has not ended, they're still responsible for voting. In other words, within the same epoch for the current active validator set, the number of blocks is always the same for all validators.

Reply:
should we check the statistic based on lifetime votes across multiple epoch, which seems more indicative of the performance?

Reply:
hmm, after reading it once more, seems performance is basically the votes in current epoch used for determining the reward in the current epoch. Please disregard my non-sense comment above.  But maybe use a different name "ValidatorEpochPerformance" or "ValidatorEpochVotes"to reduce confusion ?

Reply:
I don't feel too strongly about the name but every resource in Stake is actually scoped to an epoch. Changes to most of everything here are not effective until the next epoch, so I don't think we need to explicitly add Epoch to the names.